.github/workflows/build.yaml

```diff
@@ -1,11 +1,6 @@
 name: Build
 on:
-  push:
```

Comment:
Why remove `push`?

Reply:
I figured it best to just get rid of all of it, even on a push

Reply:
So it won't run when pushed on master. Will it run when pushing to the branch of a PR?

Reply:
All runners run all the time, except for concurrency.

Reply:
ok I'll take your word for it :D